src/mapper/pkg/resolvers/schema.helpers.resolvers.go

```diff
@@ -314,6 +315,33 @@ func (r *Resolver) handleAzureOperationReport(ctx context.Context, operation mod
 	return nil
 }
 
+func (r *Resolver) handleTrafficLevelReport(ctx context.Context, results model.TrafficLevelResults) error {
```

Comment:
unfortunately, resolving source and dest pods just by trying to search for the IP in kubefinder is not that reliable, due to annoying things like IP reuse or host-network pods. Please follow the flow performed by `handleTCPCaptureResult` to correctly resolve the local and remote identities, taking all edge cases into consideration.

Reply:
The code in handleTCPCaptureResult also doesn't work well with traffic info, since it makes assumptions on the direction of traffic. Specifically, that traffic is always from a pod to a service, which is not true in our case. I'm not sure what's the best way to handle this generally, but I hesitate to modify the code used by handleTCPCaptureResult so to not break existing features.

Reply:
Discussed offline - I'm OK with keeping this as is for this PR, and leaving an open task to improve the resolving in accordance with how handleTCPCapture works. This task should be resolved as a pre-condition to officially releasing the feature.